src/ctia/properties.clj

```diff
@@ -205,7 +205,8 @@
 
                       "ctia.store.es.event.slicing.granularity"
                       (s/enum :minute :hour :day :week :month :year)})
-   (st/optional-keys {"ctia.versions.config" s/Str})))
+   (st/optional-keys {"ctia.versions.config" s/Str})
+   (st/optional-keys {"ctia.features.disable" s/Str})))
```

Comment:
given the `(string/split x #",")` above I'd suggest:

`(s/constrained s/str (fn [s] (not (re-find #"\s" s))))`

since otherwise whitespace could create odd keywords